mlrun/alerts/alert.py

```diff
@@ -253,3 +257,103 @@ def _apply_template(self, template):
         self.criteria = self.criteria or template.criteria
         self.trigger = self.trigger or template.trigger
         self.reset_policy = self.reset_policy or template.reset_policy
+
+    def list_activations(
+        self,
+        since: Optional[datetime] = None,
+        until: Optional[datetime] = None,
+        from_last_update: bool = False,
+    ) -> list[mlrun.common.schemas.alert.AlertActivation]:
+        """
+        Retrieve a list of all alert activations.
+
+        :param since: Filters for alert activations occurring after this timestamp.
+        :param until: Filters for alert activations occurring before this timestamp.
+        :param from_last_update: If set to True, retrieves alert activations since the alert's last update time.
```

Comment:
Add info that if since and from_last_update=True is passed, from_last_update takes precedence